rdf-delta-server-local/src/main/java/org/seaborne/delta/server/local/patchstores/zk/PatchStoreZk.java

```diff
@@ -171,23 +149,12 @@ private void updateLogChanges(Set<String> names) {
     private List<String> getWatchLogs() {
         // This can happen too early when /delta/activeLogs is being setup.
         // Just ignore that and sync later.
```

Comment:
By delegating to the new `ZkConnection` instance, we get the benefit of the exception handling.